actions/plan/comment.sh

```diff
@@ -21,14 +21,17 @@ if [ "${HAS_CHANGES}" == "false" ]; then
 	exit 0
 fi
 
+TRUNCATED_WARNING=''
 PLAN_TEXT=$(terraform show "${ARTIFACTS_DIR}/terraform.plan" -no-color | sed --silent '/Terraform will perform the following actions/,$p')
 if [ "${PLAN_TEXT:0:60000}" != "${PLAN_TEXT}" ]; then
```

Comment:
Your message is a bit longer, so you might need to truncate more? Don't recall what the limit is.

Reply:
(and will continue to be every time we make it longer, until it breaks!)

Reply:
I think it's 65536, so, probably fine.